setup.py

```diff
@@ -38,6 +38,7 @@
     'gitversion',
     'APScheduler==3.0.3',
     'python-oembed == 0.2.1',
+    'html5lib<0.99999999'
```

Comment:
It is required by bleach, and a very specific version so we have to install it specifically.

env/lib/python2.7/site-packages/bleach-1.4.3.dist-info/METADATA:Requires-Dist: html5lib (>=0.999,<0.99999999)